tests/devices/default_qubit/test_default_qubit_preprocessing.py

```diff
@@ -280,6 +281,7 @@ def test_measurement_is_swapped_out(self, mp_fn, mp_cls, shots):
             (qml.GroverOperator(wires=range(14)), False),
             (qml.pow(qml.RX(1.1, 0), 3), True),
             (qml.pow(qml.RX(qml.numpy.array(1.1), 0), 3), False),
+            (qml.QubitUnitary(sp.sparse.csr_matrix(np.eye(8)), wires=range(3)), True),
```

Comment:
Am I right to assume that the stop condition applied to a dense defined `QubitUnitary` would always return True?

Reply:
For sparse defined QubitUnitary, it always returns True